openrtb-core/src/main/java/com/iabtechlab/openrtb/v2/json/OpenRtbJsonReader.java

```diff
@@ -344,6 +344,12 @@ protected void readImpField(JsonParser par, Imp.Builder imp, String fieldName)
           imp.addMetric(readMetric(par));
         }
         break;
+      case "rwwd":
```

Comment:
Typo in the original field, please change to `rwdd`.

Reply:
Please close this PR and look at this one https://github.com/IABTechLab/openrtb-proto-v2/pull/45